packages/core-web/src/styles/markbind.css

```diff
@@ -392,3 +392,14 @@ button.copy-btn {
 .navbar.navbar-dark.bg-primary .navbar-nav .current {
     background: rgba(255, 255, 255, 0.05);
 }
+
+/* Fix for Navbar dropdown menu not showing in Bootswatch v4.5.1 */
+a:not([href]) {
+    color: inherit;
```

Comment:
hmm should we move this over to the corresponding dropdown class instead?
So we don't override the changes in bootswatch (likely the purpose was to exclude user-styled links with `class`)

Reply:
Yup that makes sense. I have made the changes.